buildfox.py

```diff
@@ -902,9 +922,25 @@ def print(self, obj):
 	def assign(self, obj):
 		name = self.eval(obj[0])
 		value = obj[1]
+
+		optional_transformer = self.transformers.get(name)
+		if optional_transformer:
+			value = self.transform(optional_transformer, value)
+
 		self.variables[name] = value
 		self.output.append("%s = %s" % (name, value))
 
+	def setup_transform(self, obj):
```

Comment:
I just would change names :
- transform to eval_transform
- setup_transform to transform

But maybe we can change other names like eval, eval_path to something else instead.